cpp/include/DataStorm/InternalT.h

```diff
@@ -168,7 +172,7 @@ namespace DataStormI
         {
             std::lock_guard<std::mutex> lock(_mutex);
             std::vector<std::shared_ptr<typename V::BaseClassType>> seq;
-            for (auto& v : values)
+            for (auto v : values)
```

Comment:
I should probably revert this. The param `values` is a copy, and the previous version was moving the entries out. Which avoid the copy...